src/interfaces/IMessageDispatcher.sol

```diff
@@ -26,24 +23,25 @@ interface IMessageDispatcher {
   );
 
   /**
-   * @notice Emitted when a batch of messages has successfully been dispatched to the executor chain.
-   * @param messageId ID uniquely identifying the messages
-   * @param from Address that dispatched the messages
-   * @param toChainId ID of the chain receiving the messages
-   * @param messages Array of Message that was dispatched
+   * @notice Dispatch a message to the receiving chain.
+   * @dev Must compute and return an ID uniquely identifying the message.
+   * @dev Must emit the `MessageDispatched` event when successfully dispatched.
+   * @param toChainId ID of the receiving chain
+   * @param to Address on the receiving chain that will receive `data`
+   * @param data Data dispatched to the receiving chain
+   * @return ID uniquely identifying the message
    */
-  event MessageBatchDispatched(
-    bytes32 indexed messageId,
-    address indexed from,
-    uint256 indexed toChainId,
-    MessageLib.Message[] messages
-  );
+  function dispatchMessage(
+    uint256 toChainId,
+    address to,
+    bytes calldata data
+  ) external returns (bytes32);
 
   /**
    * @notice Retrieves address of the MessageExecutor contract on the receiving chain.
    * @dev Must revert if `toChainId` is not supported.
    * @param toChainId ID of the chain with which MessageDispatcher is communicating
-   * @return address MessageExecutor contract address
+   * @return MessageExecutor contract address
    */
   function getMessageExecutorAddress(uint256 toChainId) external returns (address);
```

Comment:
`getMessageExecutorAddress` is not required in the ERC5164 specification, so we should put it in an extended interface instead.

Reply:
Fixed in this commit: https://github.com/GenerationSoftware/ERC5164/pull/4/commits/be2d1eb98d887166ad95a2e1d88dfa9a320b23dd